### ISSUE TITLE
Discrepancy between git source and pip 0.0.21 package for convert_reshape

### ISSUE THREAD

**Issue by Jukyy** [nerox8664/onnx2keras#75]
I have tried to convert onnx model with change_ordering=True but conversion broke on convert_reshape.
I have managed to perform conversion by removing line 179 in /Lib/site-packages/onnx2keras/reshape_layers.py

	layers[node_name].set_shape([None, *input_1[1:]])

This line breaks when shape has values which are less than 0. In my conversion input_1 was [None, -1, 3]. With that line removed layers[node_name] shape was [None, None, 3] and it worked (I have tested outputs from PyTorch and Keras models and they were identical).

When I tried to post an issue for this problem, I found out that previously mentioned line is not presented in a git source (for tag v0.0.21):
https://github.com/nerox8664/onnx2keras/blob/0585e070ea96cad9f08eed3dcc7d429fe9b82dd0/onnx2keras/reshape_layers.py#L176-L179

After a further research I found out that this problem occurs only in onnx2keras version 0.0.21 but not in 0.0.20.

0.0.20 version (https://pypi.org/project/onnx2keras/0.0.20/#files) contains following lines in reshape_layers.py convert_reshape:

	reshape = keras.layers.Reshape(np.int32(input_1[1:]), name=keras_name)
	layers[node_name] = reshape(layers[node_name])

0.0.21 version (https://pypi.org/project/onnx2keras/0.0.21/#files) contains following lines in reshape_layers.py convert_reshape:

	reshape = keras.layers.Reshape(np.int32(input_1[1:]), name=keras_name)
	layers[node_name] = reshape(layers[node_name])
	print(layers[node_name], input_1)
	layers[node_name].set_shape([None, *input_1[1:]])

I am guessing that these two line were mistakenly added during debug or testing process. If that is not the case could you explain how did it came to that.

My environment:
onnx | 1.7.0 
onnx2keras | 0.0.21
python | 3.6.10
tensorflow | 2.1.0

**Comment by nerox8664** [comment]
Hello @Jukyy.

Yes, it was a temporary code I wrote when I tried to debug ONNX version of ShuffleNet model conversion and it accidentally was packed to the pip package.

Thanks for your research! I updated pip package version.

**Comment by Jukyy** [comment]
Hello @nerox8664.

Thank you for a quick fix
